src/clue/app-config.json

```diff
@@ -132,6 +124,13 @@
             "dataTestItem": "class-work-list-items",
             "documentTypes": ["publication"],
             "showStars": ["teacher"]
+          },          {
```

Comment:
The formatting is off here, but at this point I say we deal with that the next time we're in here.

Reply:
Fixed